app/views/notifier/invite.markerb

```diff
@@ -1 +1 @@
-<%= t('.message', :invite_url => invite_code_url(@invitation_code), :diasporafoundation_url => 'https://diasporafoundation.org/') %>
+<%= t('.message', :invite_url => invite_code_url(@invitation_code), :diasporafoundation_url => 'https://diasporafoundation.org/', :diaspora_id => @inviter.diaspora_handle) %>
```

Comment:
`@inviter` is not defined if you call this via `InvitationsController#email`, so you need to define it there too.

See broken spec: https://travis-ci.org/diaspora/diaspora/jobs/126729534#L591